client/src/components/VizOrbitControl/VizOrbitControl.tsx

```diff
@@ -0,0 +1,18 @@
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls';
+import { useThree } from '@react-three/fiber';
+import { useEffect } from 'react';
+
+function VizOrbitControl() {
+  const { camera, gl } = useThree();
+  useEffect(() => {
+    const controls = new OrbitControls(camera, gl.domElement);
+    controls.minDistance = 3;
+    controls.maxDistance = 20;
+    return () => {
```

Comment:
Here's one of those useEffect "cleanup" methods we briefly discussed last week: https://blog.logrocket.com/understanding-react-useeffect-cleanup-function/